packages/app/src/styles/styleFragments.ts

```diff
@@ -17,6 +17,10 @@ export const SF = makeStyleFragments({
     w: `100%`,
     h: `100%`,
   },
+  transition:{
+    transition: `all 0.5s`,
+  },
```

Comment:
it is likely that this style won't be really shared across many components. Instead, just put it inline like `<x.div transition='all 0.3s'> ...`. And I decided that the default transition time will be 0.3s.

```suggestion
```

Reply:
I removed transition style object